docs/installation.rst

```diff
@@ -22,42 +24,133 @@ Or from GitHub (for development builds)
     git clone https://github.com/PlasmaControl/DESC.git
     cd DESC
 
-    # OPTION 1: install with pip
+    # OPTION 1: install with pip after first creating a conda environment
+    conda create
     # standard build
     pip install -r requirements.txt
     # developer build (if you want to run tests)
     pip install -r devtools/dev-requirements.txt
 
     # OPTION 2: install with conda
-    # standard build
+    # only need to do ONE of these conda env create commands, not both
+    ## standard build
     conda env create --file requirements_conda.yml
-    # developer build (if you want to run tests)
+    ## developer build (if you want to run tests)
     conda env create --file devtools/dev-requirements_conda.yml
+
+
     conda activate desc-env
+    python setup.py install
```

Comment:
Unrelated, but the `setup.py` file contains the line `python_requires="~=3.7"`. Should this be changed to `>= 3.8`? The `~=` specifier means 3.7 is compatible. https://peps.python.org/pep-0440/#version-specifiers